src/protos/io/defang/v1/fabric.proto

```diff
@@ -209,6 +221,44 @@ message SecretValue {
   string project = 3; // defaults to tenant ID
 }
 
+message Config {
+  string name = 1;
+  string value = 2;
+  bool is_sensitive = 4;
+}
+
+message GetConfigsRequest {
+  repeated string names = 1;
```

Comment:
gRPC design guidelines discourage repeated primitives in favor of repeated messages

Reply:
In this case, maybe it makes sense to have a tuple (name, project) and repeat that